debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnection.java

```diff
@@ -181,6 +195,10 @@ public void getChangesForTables(SqlServerChangeTable[] changeTables, Lsn interva
             final Lsn fromLsn = getFromLsn(changeTable, intervalFromLsn);
             LOGGER.trace("Getting changes for table {} in range[{}, {}]", changeTable, fromLsn, intervalToLsn);
             preparers[idx] = statement -> {
+                String fetchSizeStr = config().asProperties().getProperty("incremental.fetch.size");
```

Comment:
We should probably consider extracting `incremental.fetch.size` up and using this across multiple connectors?

Reply:
While conducting performance tests on SQL server connector, we realized it's going with default JDBC ResultSet fetch size for incremental reading but we should give option to user configurable.  Parameterized the fetch size, thought of keeping generic but there is already snapshot.fetch.size **Field** exists hence introduced new **Field**

Reply:
I have moved this to a field specific to SQL Server called `streaming.fetch.size` to be consistent with `snapshot.fetch.size`.  We can look at moving this to common connector configurations in https://issues.redhat.com/browse/DBZ-2448.

Reply:
Thanks!